### PR TITLE
Add PUBLISHING.md

### PR DESCRIPTION
**Pull request by michaelckelly**
Closes https://github.com/plaid/plaid-node/issues/48.

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -4,3 +4,4 @@
 /.jshintrc
 /Makefile
 /circle.yml
+/PUBLISHING.md
```
Comment (davidchambers) [inline .npmignore]: Strictly speaking this should come between `/Makefile` and `/circle.yml`.

File: PUBLISHING.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Publishing
+
+The module is published to the [npm public registry][1] under the package name [plaid][2].
+
+The publish process is handled by [xyz][3] and is exposed as a [Makefile][4] target.
```
Comment (davidchambers) [inline PUBLISHING.md]: I would write _publishing process_ rather than _publish process_.

Comment (michaelckelly) [inline PUBLISHING.md]: :zap:

File: .npmignore (hunk 1 above)
Comment (michaelckelly) [inline .npmignore]: :zap:

File: PUBLISHING.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+The module is published to the [npm public registry][1] under the package name [plaid][2].
+
+The publishing process is handled by [xyz][3] and is exposed as a [Makefile][4] target.
+
+To publish:
+
+```console
+$ make release-[major|minor|patch]
+```
```
Comment (davidchambers) [inline PUBLISHING.md]: It might be worth adding:

```console
$ git checkout master
$ git pull git@github.com:plaid/plaid-node.git master
```

**Comment by davidchambers** [comment]
:deciduous_tree:

**Pull request merged**